integration-tests/gradle/src/test/resources/uber-jar-for-multi-module-project/application/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
```

Comment:
You can remove this file as it's not needed

Reply:
Okk